app/funding/get-funding.js

```diff
@@ -21,6 +21,7 @@ const requestEligibleFunding = async (organisation, callerId, correlationId) =>
   const { sbi, organisationId } = organisation
   const body = { sbi, callerId, organisationId }
 
+  console.log(`fund sbu: ${sbi} -- org ${organisationId} -- call ${callerId}`)
```

Comment:
@marctemp I wonder if replacing this line with an actual logger might be better? 
LMK your thoughts on this.
Thanks